networkx/algorithms/centrality/voterank_alg.py

```diff
@@ -90,3 +90,92 @@ def voterank(G, number_of_nodes=None):
             vote_rank[nbr][1] -= 1 / avgDegree
             vote_rank[nbr][1] = max(vote_rank[nbr][1], 0)
     return influential_nodes
+
+
+def vote_rank(G, number_of_nodes=None):
+    """Select a list of influential nodes in a graph using VoteRank algorithm
+
+    VoteRank [1]_ computes a ranking of the nodes in a graph G based on a
+    voting scheme. With VoteRank, all nodes vote for each of its in-neighbours
+    and the node with the highest votes is elected iteratively. The voting
+    ability of out-neighbors of elected nodes is decreased in subsequent turns.
+
```

Comment:
Explain in a second paragraph the difference between the two functions. You could add something to the original function too telling people to use this version if they want the vote values.